server/apiCalls/kursPmDataApi.js

```diff
@@ -15,9 +15,10 @@ async function getSortedAndPrioritizedMiniMemosWebOrPdf(courseCode) {
   try {
     const uri = client.resolve(paths.getPrioritizedWebOrPdfMemosByCourseCode.uri, { courseCode })
 
-    const { body } = await client.getAsync({ uri })
-    if (!body) log.debug('kurs-pm-data-api is not available at the moment in getSortedAndPrioritizedMiniMemosWebOrPdf')
-    return body || []
+    const res = await client.getAsync({ uri })
```

Comment:
Previous code is more readable and follows best practices. Therefore it has to be changed as it was before. Destructure direct in await, `const { body } = await client.getAsync({ uri })`

Reply:
reverted back to old code